### PR TITLE
feat!: drop support for python 2.7 and django11

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
This PR drops support for python2.7 and django11 and adds tests for python3.8 as in the other plugins

### CODE REVIEW

File: eox_hooks/serializers.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ class CertificateSerializer(serializers.ModelSerializer):
     """
     created_date = serializers.DateTimeField(format="%Y-%m-%d", read_only=True)
 
-    class Meta(object):
+    class Meta(object):  # pylint: disable=useless-object-inheritance
```
Comment (MoisesGSalas) [inline eox_hooks/serializers.py:18]: Why not:
```suggestion
    class Meta:
```
?

**Review by MoisesGSalas — COMMENTED** [review]
I think you can remove this.

https://github.com/eduNEXT/eox-hooks/blob/06707e4dcbe496264973cea5dda3848ae132e3ee/eox_hooks/views.py#L6

https://github.com/eduNEXT/eox-hooks/blob/06707e4dcbe496264973cea5dda3848ae132e3ee/eox_hooks/__init__.py#L5

https://github.com/eduNEXT/eox-hooks/blob/06707e4dcbe496264973cea5dda3848ae132e3ee/eox_hooks/apps.py#L5

https://github.com/eduNEXT/eox-hooks/blob/06707e4dcbe496264973cea5dda3848ae132e3ee/eox_hooks/settings/common.py#L11

https://github.com/eduNEXT/eox-hooks/blob/06707e4dcbe496264973cea5dda3848ae132e3ee/eox_hooks/settings/production.py#L5

https://github.com/eduNEXT/eox-hooks/blob/06707e4dcbe496264973cea5dda3848ae132e3ee/eox_hooks/settings/test.py#L5

**Comment by mariajgrimaldi** [comment]
@MoisesGSalas yes! Thanks

File: eox_hooks/edxapp_wrapper/backends/models.py [hunk 2]
```diff
@@ -1,13 +1,14 @@
 """
 Simple backend that returns the platform's models
 """
+# pylint: disable=import-error,import-outside-toplevel
```
Comment (andrey-canon) [inline eox_hooks/edxapp_wrapper/backends/models.py:4]: this disable everything and that is a thing that we don't want to do

Comment (mariajgrimaldi) [reply eox_hooks/edxapp_wrapper/backends/models.py:4]: In a backend? Where we're going to have a lot of import errors and import-outside-top-level?

**Review by MoisesGSalas — APPROVED** [review]

File: eox_hooks/edxapp_wrapper/backends/models.py (hunk 2 above)
Comment (andrey-canon) [reply eox_hooks/edxapp_wrapper/backends/models.py:4]: it concerns me that we won't catch new errors

**Pull request merged**